www/multiview.js

```diff
@@ -2,16 +2,17 @@
 
 var exec = require('cordova/exec');
 var urlutil = require('cordova/urlutil');
-
 module.exports = {
+    loadView:function(url, message, success, error) {
+        strPath = urlutil.makeAbsolute(url);
```

Comment:
Careful here, because you changed the loadView param from `strPath` to `url` this means that `strPath` is technically undefined here and should be defined with `var strPath;`